src/components/ui/PaymentRequestRow/PaymentRequestRow.tsx

```diff
@@ -27,39 +30,101 @@ const Row = ({
   onCopyLink,
   onDelete,
 }: PaymentRequestRowProps) => {
+  const [isDeleting, setIsDeleting] = useState(false);
+  const rowRef = useRef<HTMLTableRowElement>(null);
+  const overlayRowRef = useRef<HTMLTableRowElement>(null);
+
+  useEffect(() => {
+    console.log('CALLED ' + id);
+    if (rowRef.current && overlayRowRef.current && isDeleting) {
+      let { top } = rowRef.current.getBoundingClientRect();
+      top += window.scrollY;
+      top -= 6; // 6px padding
+      overlayRowRef.current.setAttribute('style', `top: ${top}px`);
+    }
+  }, [isDeleting]);
+
+  const onDeletePaymentRequestClicked = async () => {
+    setIsDeleting(true);
+    animate(`tr.custom-backdrop-blur-${id}`, { opacity: 0.2 }, { duration: 0.2 });
+  };
+
+  const onCancelDeletingPaymentRequestClicked = async () => {
+    setIsDeleting(false);
+    animate(`tr.custom-backdrop-blur-${id}`, { opacity: 1 }, { duration: 0.2 });
+  };
```

Comment:
Not sure how difficult would this be, but could it be possible to automatically trigger a cancel deleting payment request when clicking outside the row the user wanted to delete?

Don't want to make this ultra complex it's just that I found myself kind of having this thought.